charactersheet/charactersheet/models/common/weapon.js

```diff
@@ -136,6 +136,27 @@ function Weapon() {
         }
     });
 
+    self.weaponRangeLabel = ko.pureComputed(function() {
+        if (self.weaponType().toLowerCase() === 'ranged'){
```

Comment:
Honestly I prefer to be explicit when checking in a situation like that.  I think it's more extensible that way.